dictionaries/people-names/dict/people-names.txt

```diff
@@ -1,5 +1,6 @@
 
 # cspell-tools: keep-case no-split
+# cspell-dictionary: no-generate-alternatives
```

Comment:
This line tell the spell checker to not generate lower case versions of the words.